app/views/browse/build-config.html

```diff
@@ -78,20 +79,22 @@
                     <div ng-if="unfilteredBuilds && (unfilteredBuilds | hashSize) === 0" class="empty-state-message text-center">
                       <h2>No builds.</h2>
 
-                      <p>
-                        Start a new build to create an image from
-                        <span ng-if="buildConfig.spec.source.type === 'Git'">
-                          source repository
-                          <span class="word-break" ng-bind-html='buildConfig.spec.source.git.uri | githubLink : buildConfig.spec.source.git.ref : buildConfig.spec.source.contextDir | linky'></span>
-                        </span>
-                        <span ng-if="buildConfig.spec.source.type !== 'Git'">
-                          build configuration {{buildConfig.metadata.name}}.
-                        </span>
-                      </p>
+                      <div ng-if="'buildconfigs/instantiate' | canI : 'create'">
+                        <p>
+                          Start a new build to create an image from
```

Comment:
instead of hiding this entire block, i would just tweak this text a bit, and only hide the start button when the user can't instantiate builds.  maybe make the text 'New builds will create an image from ....'  or even just 'Builds will create an image from...'